iac-extensions/docker/src/main/java/org/sonar/iac/docker/utils/CheckUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.docker.utils;
+
+import java.util.List;
+import java.util.Optional;
+import org.sonar.iac.docker.tree.api.ParamTree;
+
+public class CheckUtils {
+
+  private CheckUtils() {
+  }
+
+  public static Optional<ParamTree> getParamByName(List<ParamTree> params, String name) {
```

Comment:
I would suggest creating a wrapper for `List<ParamTree>`, adding this method there, and replacing all usages of `List<ParamTree> options` like this:

```
class OptionsTree ... {
  private List<ParamTree> options;
  public Optional<ParamTree> getParamByName(String name) { ... }
}
```

This will improve the design and we can avoid *Utils classes. 
This may be done with a separate ticket/PR.
WDYT?